python/ql/src/experimental/semmle/python/Concepts.qll

```diff
@@ -13,3 +13,70 @@ private import semmle.python.dataflow.new.DataFlow
 private import semmle.python.dataflow.new.RemoteFlowSources
 private import semmle.python.dataflow.new.TaintTracking
 private import experimental.semmle.python.Frameworks
+
+/** Provides classes for modeling Regular Expression-related APIs. */
+module RegexExecution {
+  /**
+   * A data-flow node that collects methods immediately executing an expression.
```

Comment:
```suggestion
   * A data-flow node that executes a regular expression.
```
While the class `RegexExecution` collects data flow nodes with this description, I think we here want to describe directly what makes a given data flow node interesting.